reconcile/github_org.py

```diff
@@ -499,3 +499,10 @@ def run(dry_run):
     )
 
     runner.run()
+
+
+def early_exit_desired_state(*args, **kwargs) -> dict[str, Any]:
+    return {
+        "github_org_config": get_config(),
+        "github_org_members": fetch_desired_state().dump(),
```

Comment:
wouldn't you prefer to replace this with `permissions` and `clusters`?

Reply:
queried `clusters` are intertwined with `openshift_users.fetch_desired_state(oc_map=None)` so this processing done in `fetch_desired_state` seems to meaningful for getting the picture about the desired state.